### ISSUE TITLE
Feature Request: Ctrl+D duplicate object

### ISSUE THREAD

**Issue by randyyaj** [evolus/pencil#314]
* Operating system: Mac
* Pencil version: Latest

Be able to hit cmd+d or ctrl+d to duplicate an object.

**Comment by dgthanhan** [comment]
In case you don't know, holding Shift while dragging an object will duplicate it.

**Comment by randyyaj** [comment]
Yeah I figured that one out shortly. I was just thinking in terms of how other programs do it.
